build.gradle

```diff
@@ -71,7 +71,7 @@ ext {
     noticeFile = rootProject.file('NOTICE.txt')
 }
 
-sourceCompatibility = 1.9
+sourceCompatibility = targetCompatibility = "1.8"
```

Comment:
Super minor but maybe more conventional: 
```
sourceCompatibility = 1.8
targetCompatibility = 1.8
```

Reply:
We kinda do both, but yes.